### DISCUSSION TITLE
2.6.0 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#1819] [讨论]
· 支持在多个正版账号间切换
· Mod / 整合包下载会单独列出筛选的版本（#1755，@infidiotWolf）
· 崩溃分析优化，支持更多原因的崩溃分析（#1773，by @zkitefly）
· 由于官网锁区，购买正版的链接将跳转到 XBox（#1806，@deep-dark-forest）
· 细节优化
　　修改正版账户功能按钮的样式，增强可见性（#1719，@xitieshiz2）
　　为音乐音量调节添加了百分比显示（#1727，@MNWZ211）
　　在检查更新、无需更新时显示版本号（#1739，@Bandysol）
　　加高 JVM 参数的输入框（#1763，@iPodtouch6557）
　　为背景图片、音乐数量添加单位（#1775，@iPodtouch6557）
　　为滚动条添加进入、退出动画（#1779，@youzi-2333）
　　优化 LittleSkin 设置覆盖提示（#1787，@youzi-2333）
　　音乐播放允许选择顺序播放或随机播放（#1816，@iPodtouch6557）
· 修复了以下 2 个 Bug
　　修改玩家正版用户名的链接指向英文网站（#1772，by @zkitefly）
　　部分“你知道吗？”条目已经过时（#1808，@TheXiaoma）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.0'>2.6.0</a>.</em>

**Comment by iPodtouch6557** [comment]
三个我欸😆

**Reply by MNWZ211** [reply]
![-276b3ae2cdba08e2.jpg](https://github.com/Hex-Dragon/PCL2/assets/80450278/1a52efea-e68e-48c2-ae15-df883452fa8c)
